.gitignore

```diff
@@ -10,3 +10,28 @@ packages/*/*.js
 packages/*/bin
 !packages/*/.eslintrc.js
 *.log
+
+#TS output
+packages/**/dist
+dist
+build
+
+# Debug log from npm
+**/npm-debug.log
+
+# Debug log from lerna
+lerna-debug.log
+
+# Debug logs from yarn
+**/yarn-error.log
+
+# IntelliJ IDE ignore
+.idea
+
+tmp
+**/test.log
+
+tsconfig.tsbuildinfo
+tsconfig.prod.tsbuildinfo
```

Comment:
Does this impact the `@sewing-kit` package at all?

Reply:
I don't think so - `.tsbuildinfo` will be recreated if it doesn't exist. It shouldn't be checked in since we want to do a fresh build when making a release

Reply:
Why are these going to the root of the project?